playwright/tests/challenges.spec.ts

```diff
@@ -213,8 +216,20 @@ test.describe("Hacker challenge speedrun", () => {
     });
     await expect(challengeNinePage.passwordImage).toBeVisible();
 
+    // take screenshot from website
+    await page.screenshot({ path: 'revealed_image_with_password.png' });
+
+    // use Tesseract worker to recognize the text that is displayed on the website
+    // https://github.com/tesseract-ocr/tesseract
+    const { data: { text } } = await (await worker).recognize('revealed_image_with_password.png');
+    console.log('### Recognized text on website:\n', text);
```

Comment:
Since `getStatedPasswordIn` might return an empty string I'd suggest to add an assertion that the extracted password is not empty like:
```suggestion
    console.log('### Recognized text on website:\n', text);
    await expect(text).not.toBeEmpty()
```

Reply:
I got a type error when using `toBeEmpty()`, this is why I went with `toEqual('')`